repondeur/zam_repondeur/templates/lectures_add.html

```diff
@@ -12,56 +18,73 @@ <h2>Ajouter une lecture...</h2>
 
     <form action="{{ request.path }}" method="POST">
         <div class="form-group">
-            <label for="chambre">Chambre</label>
-            <select name="chambre" class="form-control">
-                {% for id, label in chambres %}
-                    <option value="{{ id }}" {%- if loop.first %} selected{% endif %}>{{ label }}</option>
+            <label for="dossier">Texte de loi</label>
+            <select id="select-dossier" name="dossier" class="form-control">
+                {% for dossier in dossiers %}
+                    <option value="{{ dossier.uid }}" {%- if loop.first %} selected{% endif %}>{{ dossier.titre }}</option>
                 {% endfor %}
             </select>
         </div>
         <div class="form-group">
-            <label for="session">Session</label>
-            <select name="session" class="form-control">
-                {% for id, label in sessions %}
-                    <option value="{{ id }}" {%- if loop.first %} selected{% endif %}>{{ label }}</option>
+            <label for="lecture">Lecture</label>
+            <select id="select-lecture" name="lecture" class="form-control">
+                {% for texte_uid, titre in lectures_by_dossier[dossiers[0].uid].items() %}
+                    <option value="{{ texte_uid }}" {%- if loop.first %} selected{% endif %}>{{ titre }}</option>
                 {% endfor %}
             </select>
         </div>
-        <div class="form-group">
-            <label for="num_texte">Numéro de texte</label>
-            <input type="number" name="num_texte" value="63" class="form-control">
-        </div>
         <div class="form-group">
             <input type="submit" name="submit" value="Ajouter" class="btn btn-primary btn-lg">
             <a href="{{ request.route_url('lectures_list') }}" class="btn">Annuler</a>
         </div>
     </form>
 
-    {% if amendements %}
-    <table class="table table-striped">
-        <thead>
-            <tr>
-                <th>Article</th>
-                <th>Alinéa</th>
-                <th>Numéro</th>
-                <th>Auteur</th>
-                <th>Groupe</th>
-                <th>Sort</th>
-            </tr>
-        </thead>
-
-        <tbody>
-            {% for amendement in amendements %}
-                <tr>
-                    <td>{{ amendement.article }}</td>
-                    <td>{{ amendement.alinea }}</td>
-                    <td>{{ amendement.num }}</td>
-                    <td>{{ amendement.auteur }}</td>
-                    <td>{{ amendement.groupe }}</td>
-                    <td>{{ amendement.sort }}</td>
-                </tr>
-            {% endfor %}
-        </tbody>
-    </table>
-    {% endif %}
+{% endblock %}
+
+
+{% block scripts %}
+<script src="{{ request.static_url('zam_repondeur:static/choices/choices.min.js') }}"></script>
+<script type="text/javascript">
+
+    var lectures_by_dossier = {{ lectures_by_dossier | tojson | safe }};
```

Comment:
Not sure where we want to go here with ES6/semicolumn-less/etc.

Reply:
I'm not sure what the best practices are today. Do all important browsers support ES6 now?

(I think I do like semicolons though, must be all the time I spent writing C and C++?)